cob_moveit_config/raw3-1/config/kinematics.yaml

```diff
@@ -1,16 +1,11 @@
 arm:
   kinematics_solver: kdl_kinematics_plugin/KDLKinematicsPlugin
-  #kinematics_solver: cob_kinematics/IKFast_ur10
   kinematics_solver_search_resolution: 0.005
-  kinematics_solver_timeout: 0.005
+  kinematics_solver_timeout: 5
```

Comment:
5 seconds for an IK solution?
You could use `ur_kinematics/UR10KinematicsPlugin` it's an analytic solver rather than the numerical KDL solver